dataretrieval/samples.py

```diff
@@ -1,233 +1,136 @@
-"""
-Tool for downloading data from the USGS Aquarius Samples database (https://waterqualitydata.us)
+"""Functions for downloading data from the USGS Aquarius Samples database
+(https://waterdata.usgs.gov/download-samples/#dataProfile=site).
```

Comment:
Thanks for correcting this. I actually think it can go one step cleaner:
```suggestion
(https://waterdata.usgs.gov/download-samples/).
```